src/__tests__/utils.spec.ts

```diff
@@ -7,6 +7,9 @@ const fixtureVersions = ['0.59', '0.58', '0.57', '0.56'].map((version) => ({
   createApp: '',
```

Comment:
mmm I think this will do the trick, so that we stay more aligned with the upstream project

```suggestion
  createApp: undefined!,
```